karma.conf.js

```diff
@@ -89,7 +89,15 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['Chrome_Sized'],
+
+
+    customLaunchers: {
+      'Chrome_Sized': {
+        base: 'Chrome',
+        flags: ['--window-size=400,600']
```

Comment:
Sounds good. If you can't get it to work, just open up an issue to keep
track of what remains

On Mon, May 16, 2016 at 10:46 AM, Stewart Yu <notifications@github.com>
wrote:

> In karma.conf.js
> <https://github.com/CasperSleep/nightshade-core/pull/131#discussion_r63365179>
> :
>
> > @@ -89,7 +89,15 @@ module.exports = function(config) {
> >
> >      // start these browsers
> >      // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
> > -    browsers: ['PhantomJS'],
> > +    browsers: ['Chrome_Sized'],
> > +
> > +
> > +    customLaunchers: {
> > +      'Chrome_Sized': {
> > +        base: 'Chrome',
> > +        flags: ['--window-size=400,600']
>
> @misscs <https://github.com/misscs> It does - I just couldn't get it to
> work. Let me try again though because I do agree that would be the best way
> to go.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/CasperSleep/nightshade-core/pull/131/files/ad9908c50fbd28283712162ad3ce2845f6cbef74#r63365179>
>